src/toil/wdl/wdltoil.py

```diff
@@ -1579,7 +1580,7 @@ def run(self, file_store: AbstractFileStore) -> Promised[WDLBindings]:
             total_bytes: float = convert_units(total_gb, 'GB')
             runtime_disk = int(total_bytes)
 
-        if runtime_bindings.has_binding('gpuType') or runtime_bindings.has_binding('gpuCount') or runtime_bindings.has_binding('nvidiaDriverVersion'):
+        if runtime_bindings.has_binding('gpu') or runtime_bindings.has_binding('gpuType') or runtime_bindings.has_binding('gpuCount') or runtime_bindings.has_binding('nvidiaDriverVersion'):
```

Comment:
We don't actually check the value of the `gpu` key, and we default `gpuCount` to 1 if it is not set. So if I have a task that just has `gpu: false` and none of the other keys, it's going to get 1 GPU.

We probably should check the `gpu` key value, and if it's present and explicitly false we should let it override the other ones, since it's the standard one. I could see someone writing a workflow where `gpuCount` is always 1 and `gpu` is determined by some expression and meant to gate whether GPU mode is actually used.